ruby/rails/rubocop-disabled.yml

```diff
@@ -34,6 +34,18 @@ Style/Documentation:
 Style/DotPosition:
   Enabled: false
 
+# Allow formatting like
+#
+#   return some_value if some_really_long_condition_that_eats_up_the_line ||
+#     another_pretty_long_condition_that_would_not_be_readable_if_further_indented
+#
+# instead of requiring formatting like
+#
+#   return some_value if some_really_long_condition_that_eats_up_the_line ||
+#                        another_pretty_long_condition_that_would_not_be_readable_if_further_indented
+Style/MultilineOperationIndentation:
+  Enabled: false
```

Comment:
Sometimes Rubocop gives you and intermediate option where it accepts several styles as valid and none others. I'd like either aligned or two space indent and that is all. Is there something like this?

Reply:
It looks to me like there are only [two options](https://github.com/bbatsov/rubocop/blob/master/manual/cops_layout.md#layoutmultilineoperationindentation) (`aligned` and `indented`) neither of which allows what I want to allow, even though the latter sounded promising at first.

Reply:
I think `indented` is what you want. The example on the page does not convey it well.

Reply:
Is there an easy way to test your assertion? I could dig through Rubocop docs, but I figure you already know how.

Reply:
(Sorry I'm not familiar with running the linter myself.)

Reply:
`codeclimate prepare` will pull down the files. `rubocop file_pattern` will run rubocop.